### DISCUSSION TITLE
What to work on?

### DISCUSSION THREAD

**Discussion by boxydog** [getpelican/pelican#3319] [General]
@justinmayer

Any thoughts about the highest priority items for people to work on? How can we help?

**Comment by justinmayer** [comment]
Many thanks for asking, @boxydog. Much appreciated! 🙏 

Off the top of my head, the highest-priority item I can think of would be ensuring that a user who tries to build a site containing a Markdown source file, but who does not have the `Markdown` package installed, is presented with a friendly message explaining the situation and what they need to do to fix it. See existing issue: #1868

Beyond that, I would welcome input on this topic from @getpelican/reviewers and any other members of the Pelican community. What do y'all think are the highest-priority items at hand?

**Comment by boxydog** [comment]
Other related questions that might stir this pot:

- What is a "theme" that pelican should work on? Reliability? Ease of use? Growing the user base?
- Are there things that would increase the user base a lot?
- Are there problems users encounter often?
- What can we learn from other popular static site generators (e.g., hugo, jekyll)?
